src/components/postgresConnect.ts

```diff
@@ -359,6 +359,20 @@ class PostgresDB {
     }
   }
 
+  public checkIfInFileTable = async (subPath: string) => {
+    const text = `
+    SELECT id 
+    FROM file
+    WHERE file_path = '${subPath}'
+    `
```

Comment:
Please use parameterised queries:
```
SELECT id 
    FROM file
    WHERE file_path = $1
```

See the other database methods for how it works (needs a "values" parameter in `this.query...`).

Also -- this give a brief explanation as to why: https://node-postgres.com/features/queries#parameterized-query